actions/check-thirdparty-notice/check_thirdparty_notice.py

```diff
@@ -0,0 +1,120 @@
+# SPDX-FileCopyrightText: Copyright (C) 2025 Opal Health Informatics Group <https://www.opalmedapps.com>
+#
+# SPDX-License-Identifier: MIT
+
+# TODO: let Renovate update these dependencies somehow
```

Comment:
This should be supported now. Although it seems that we need to update the file matching: https://docs.renovatebot.com/modules/manager/pep723/